src/Microsoft.AspNetCore.Server.Kestrel.Https/Properties/AssemblyInfo.cs

```diff
@@ -4,8 +4,4 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: NeutralResourcesLanguage("en-us")]
-[assembly: AssemblyCompany("Microsoft Corporation.")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
-[assembly: AssemblyProduct("Microsoft ASP.NET Core")]
+[assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
This will be generated automatically by Internal.AspNetCore.Sdk. This whole file can be removed.